db/migrate/20240420012042_add_external_course_id_to_courses.rb

```diff
@@ -0,0 +1,5 @@
+class AddExternalCourseIdToCourses < ActiveRecord::Migration[7.1]
+  def change
+    add_column :courses, :external_course_id, :string
+  end
+end
```

Comment:
We should put this `external_course_id` in the `Course_to_lms` table, because the general `course` doesn't belong to any `lms`.

Reply:
Good catch, thank you for finding that!